### PR TITLE
feat(console): update docs for blueprints

### PR DESCRIPTION
**Pull request by Nuttymoon**
### Dependencies

- https://github.com/AshAvalanche/jeeo/pull/109
- https://github.com/AshAvalanche/ash-rs/pull/79

### Changes

- Console docs
  - Add the `Console Blueprint` page in the reference section
  - Updates the reference pages + guides to reflect the latest CLI changes, esp. YAML support + blueprints

### CODE REVIEW

File: docs/console/reference/blueprints.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+  secrets:
+    - name: aws-credentials
+      secretType: awsCredentials
+      accessKey: "${AWS_ACCESS_KEY}"
+      secretKey: "${AWS_SECRET_KEY}"
+  ```
+
+## Apply a blueprint
+
+Applyin a blueprint is as simple as running the `blueprint apply` command and providing the blueprint path:
```
Comment (leopaul36) [inline docs/console/reference/blueprints.md:66]:
```suggestion
Applying a blueprint is as simple as running the `blueprint apply` command and providing the blueprint path:
```

File: docs/console/guides/local-network/node-ids.md [hunk 2]
```diff
@@ -18,9 +18,13 @@ In this part of the guide, we will create **5 node ID secrets** for the 5 nodes
...
 
-## Fetch the 5 node IDs certificates
+## Fetch the node IDs blueprint
 
-The [Local Test Network Creation tutorial](/docs/toolkit/ansible-avalanche-collection/tutorials/local-test-network) uses the same certificates and keys for the 5 validator nodes. Let's fetch the certificate and key files from the [`ansible-avalanche-getting-started`](https://github.com/AshAvalanche/ansible-avalanche-getting-started) repository.
+:::info
+Learn more about blueprints in the [Console Blueprints](/docs/console/reference/blueprints) reference.
+:::
+
+We will use the [local-node-ids.yaml blueprint](https://github.com/AshAvalanche/ash-rs/blob/ash-console-alpha/crates/ash_cli/examples/console/blueprint/local-node-ids.yaml) to create the node ID secrets for the 5 nodes of our Avalanche devnet.
```
Comment (leopaul36) [inline docs/console/guides/local-network/node-ids.md:27]:
```suggestion
We will use the [local-node-ids.yaml](https://github.com/AshAvalanche/ash-rs/blob/ash-console-alpha/crates/ash_cli/examples/console/blueprint/local-node-ids.yaml) blueprint to create the node ID secrets for the 5 nodes of our Avalanche devnet.
```

File: docs/console/guides/local-network/avalanche-nodes.md [hunk 3]
```diff
@@ -10,38 +10,79 @@ The Ash Console is currently in alpha and **not production-ready**. It is under
 
 We can finally deploy our 5 Avalanche node [resources](/docs/console/glossary#resource)!
 
+## Set shared configuration at the project level
+
+It is common that **resources of the same type share some configuration**. For example, all the Avalanche nodes of our devnet will have the **same AvalancheGo version**.
```
Comment (leopaul36) [inline docs/console/guides/local-network/avalanche-nodes.md:15]:
```suggestion
It is common that **resources of the same project share some configuration**. For example, all the Avalanche nodes of our devnet will have the **same AvalancheGo version**.
```

**Review by leopaul36 — APPROVED** [review]

File: docs/console/guides/local-network/avalanche-nodes.md [hunk 4]
```diff
@@ -50,63 +91,55 @@ We can finally deploy our 5 Avalanche node [resources](/docs/console/glossary#re
...
+      isBootstrapNode: false
+    },
+    nodeBootstrapResourceId: avalanche-node-01
+  }"
+done
+```
 
 :::caution
+
 - The above command takes about ~30 seconds to complete.
```
Comment (leopaul36) [inline docs/console/guides/local-network/avalanche-nodes.md:116]: Closer to a minute

Comment (Nuttymoon) [inline docs/console/guides/local-network/avalanche-nodes.md:116]:
```suggestion
- The above command takes about 1 minute to complete.
```

File: docs/console/guides/blueprint/blueprint-configuration.md [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+  1 to create:
+  - 'ash-devnet':
+      Regions: aws/us-east-1
+      Resources: local-node-01, local-node-02, local-node-03, local-node-04, local-node-05
+  0 to update:
+? Are you sure you want to apply this blueprint? (y/N)
+[This action is irreversible!]
+```
+
+Enter `y` and watch the magic happen!
```
Comment (leopaul36) [inline docs/console/guides/blueprint/blueprint-configuration.md:144]: I think we should also copy the section from `/docs/console/guides/local-network/avalanche-nodes#health`

**Review by leopaul36 — CHANGES_REQUESTED** [review]

**Pull request merged**